druid/src/lib.rs

```diff
@@ -142,6 +142,9 @@ pub use druid_shell::{kurbo, piet};
 #[doc(inline)]
 pub use im;
 
+#[macro_use]
+pub mod lens;
```

Comment:
This was from when I was using the lens! macro (before I had generic lens derivation?). You need this in order for macro_rules things to work in the same crate. 

I left it because it might be a headache for someone else to find out how to sort if they need to use that lens! macro